test/models/food_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class FoodTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
[optional] Delete the test folder as you will not be making use of it in the project, remember that you will make use of RSpec and not the default rails test